src/clinical/service-worker-thread/runner.ts

```diff
@@ -0,0 +1,17 @@
+import { StaticPool } from 'node-worker-threads-pool';
+import { WorkerTasks } from './tasks';
+
+const poolSize = 1;
+const poolTimeOutMs = 20 * 1000;
```

Comment:
willl you wait 20 seconds for a download ?

Reply:
This if the worker thread function never returns anything, so the timeout will let us know to something went wrong. I guess this should be configurable too eh. I picked 20secs arbitrarily.

Reply:
I see I guess it's fine for now